### PR TITLE
CHANGE(netdata): Add security limits

### PR DESCRIPTION
**Pull request by GeoffreyTien**
##### SUMMARY

- add new variables to set open files limit used by netdata
- increase The default operating system limits on mmap
- change check lib zookeeper on command.conf

 ##### IMPACT

 ##### ADDITIONAL INFORMATION

### CODE REVIEW

File: files/commands/RedHat.conf [hunk 1]
```diff
@@ -4,7 +4,7 @@ swift_version=rpm -q --qf "%{VERSION}\n" openio-sds-swift
 s3_version=rpm -q --qf "%{VERSION}\n" openio-sds-swift-plugin-swift3
 redis_version=redis-server --version | grep -oP ' v=\K.+? '
 zk_version=rpm -q --qf "%{VERSION}\n" zookeeper
-zk_lib_version=rpm -q --qf "%{VERSION}\n" zookeeper-lib
+zk_lib_version=rpm -q --qf "%{VERSION}\n" libzookeeper2
```
Comment (vdombrovski) [inline files/commands/RedHat.conf]: Out of scope for PR

File: tasks/main.yml [hunk 2]
```diff
@@ -92,6 +92,26 @@
...
+  tags: configure
+
+- name: Set operating system limits on mmap  
+  template:
+    src: "openio-sysctl-netdata.conf.j2"
+    dest: "/etc/sysctl.d/netdata.conf"
+    owner: "root"
+    group: "root"
+    mode: 0644
+  when: inventory_hostname in openio_netdata_oio_hosts
```
Comment (vdombrovski) [inline tasks/main.yml]: This needs to be applied everywhere, not only on OpenIO hosts

File: tasks/main.yml [hunk 3]
```diff
@@ -92,6 +92,26 @@
...
   when: inventory_hostname in openio_netdata_oio_hosts
 
+- name: Set open files limit non-systemd 
+  template:
+    src: "openio-security-limits.conf.j2"
+    dest: "/etc/security/limits.d/30-netdata.conf"
+    owner: "root"
+    group: "root"
+    mode: 0644
+  when: inventory_hostname in openio_netdata_oio_hosts
```
Comment (vdombrovski) [inline tasks/main.yml]: Same as below

File: tasks/main.yml (hunk 2 above)
Comment (vdombrovski) [reply tasks/main.yml]: Also, don't forget to reload sysctl when modifying the configuration. In fact, you can use sysctl module to set these parameters like this:

```
sysctl:
        sysctl_file: "/etc/sysctl.d/netdata.conf"
        name: "{{ item.key }}"
        value: "{{ item.value }}"
        sysctl_set: true
        reload: true
```

**Review by vdombrovski — CHANGES_REQUESTED** [review]

File: files/commands/RedHat.conf (hunk 1 above)
Comment (vdombrovski) [reply files/commands/RedHat.conf]: See #69

File: tasks/main.yml [hunk 4]
```diff
@@ -92,6 +92,24 @@
...
   when: inventory_hostname in openio_netdata_oio_hosts
 
+- name: Set open files limit non-systemd 
+  template:
+    src: "openio-security-limits.conf.j2"
+    dest: "/etc/security/limits.d/30-netdata.conf"
+    owner: "root"
+    group: "root"
+    mode: 0644
+  tags: configure
```
Comment (cdelgehier) [inline tasks/main.yml:102]: `when: openio_security_limits_managed`

File: tasks/main.yml [hunk 5]
```diff
@@ -92,6 +92,24 @@
...
+
+- name: Set operating system limits on mmap  
+  template:
+    src: "openio-sysctl-netdata.conf.j2"
+    dest: "/etc/sysctl.d/netdata.conf"
+    owner: "root"
+    group: "root"
+    mode: 0644
+  tags: configure
```
Comment (cdelgehier) [inline tasks/main.yml:111]: `when: openio_sysctl_managed`

File: templates/python.d.conf.j2 [hunk 6]
```diff
@@ -24,3 +24,4 @@ default_run: {{ openio_netdata_python_d_plugin_default_run }}
 beanstalk: yes
 redis: yes
 web_log: {{ openio_netdata_python_d_plugin_web_log }}
+
```
Comment (cdelgehier) [inline templates/python.d.conf.j2:27]: ?

File: files/commands/RedHat.conf [hunk 7]
```diff
@@ -4,7 +4,7 @@ swift_version=rpm -q --qf "%{VERSION}\n" openio-sds-swift
 s3_version=rpm -q --qf "%{VERSION}\n" openio-sds-swift-plugin-swift3
 redis_version=redis-server --version | grep -oP ' v=\K.+? '
 zk_version=rpm -q --qf "%{VERSION}\n" zookeeper
-zk_lib_version=rpm -q --qf "%{VERSION}\n" zookeeper-lib
+zk_lib_version=rpm -q --qf "%{VERSION}\n" zookeeper-lib
```
Comment (cdelgehier) [inline files/commands/RedHat.conf:7]: ?

**Review by cdelgehier — CHANGES_REQUESTED** [review]

**Pull request closed without merging**